docs/projects/git-best-practices.ipynb

```diff
@@ -30,7 +30,10 @@
    "source": [
```

Comment:
<p>need to fix the link - &nbsp;<a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#load-code-from-container-vs-load-code-at-runtime" target="_blank">Loading the code from container vs. loading the code at runtime</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
fixed